tests/satosa/backends/test_saml2.py

```diff
@@ -226,32 +242,6 @@ def test_authn_request(self, context, idp_conf):
         req_params = dict(parse_qsl(urlparse(resp.message).query))
         assert context.state[self.samlbackend.name]["relay_state"] == req_params["RelayState"]
 
-    def test_mirror_saml_forceauthn(self, context, sp_conf):
```

Comment:
This test needs to be restored. It can now be done through `get_force_authn` function.